pkg/cvo/sync_worker.go

```diff
@@ -928,6 +933,15 @@ func newClusterOperatorsNotAvailable(errs []error) error {
 		if len(uErr.Name) > 0 {
 			names = append(names, uErr.Name)
 		}
+		switch uErr.UpdateEffect {
+		case payload.UpdateEffectNone:
+		case payload.UpdateEffectFail:
+			updateEffect = payload.UpdateEffectFail
+		case payload.UpdateEffectFailAfterInterval:
+			if updateEffect != payload.UpdateEffectFail {
+				updateEffect = payload.UpdateEffectFailAfterInterval
```

Comment:
This collapsing is going to slightly break the effect.  For example, say in round one, operator A was `UpdateEffectFailAfterInterval`.  In round two, A is `UpdateEffectNone`, but B is `UpdateEffectFailAfterInterval`.  This code collapses them into a single error that names A and B and has `UpdateEffectFailAfterInterval`.  The consuming code sees `UpdateEffectFailAfterInterval`, sees that A has been in the doghouse since the first round, and maybe goes `Failing=True` if it's been long enough.  But still, better than what we have today, and we can always tune more in the future.